### DISCUSSION TITLE
Why isn't the Application layer in a separate project?

### DISCUSSION THREAD

**Discussion by rodion-m** [dotnet/eShop#141] [Q&A]
I've been researching this solution and have a few questions about it. The first question - why isn't the Application layer in a separate project? Shouldn't the commands be isolated in the Application layer? At the moment they are in the Presentation layer (API). 

And also, is there any chat where I can ask further questions about this repo?

Thanks in advance for the explanations :)

**Comment by punkouter26** [comment]
i find the microservice examples combine them since you already have 10 or so projects as it is. and the main question to ask is what situation would you really benefit if they were put in a separate project ?
